### PR TITLE
docs: prevent shifting icon in Collapsible

### LINKED ISSUE #917: Collapsible component example (story) icon position broken for expand/collapse state toggles.
## Summary

The icon in `Collapsible` component example (story) is shifting downward/upward on expand/collapse of the collapsible container.

Expected: 
- While hovering over the Collapsible or expanding by clicking on expand button on bottom the calendar icon should not shift its vertical position.

### Motivation

The design should be consistence in case of documentation also.

## Steps to resolve this issue

- Open the story : https://innovaccer.github.io/design-system/?path=/docs/components-collapsible-all--all
- Inspect the css of the components, add or remove some helper classes as per need.

❕ This issue is under Documentation category despite there are code changes as it does not affects underline library code or features.

### PR DESCRIPTION
**Pull request by AlithAnar**
### What does this implement/fix? Explain your changes.

This fix adds "Text--regular" class to the icon className to align the appearing text line height. Before the fix a text that appears after expanding the panel increased the height of the header row and moved the icon a little bit.
...

### Does this close any currently open issues?
fixes #917
...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/atoms/collapsible/__stories__/index.story.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const all = () => {
   return (
     <div className="d-flex">
       <Collapsible expanded={expanded} hoverable={hoverable} height="100vh" onToggle={setExpanded}>
-        <div className="d-flex pt-4">
+        <div className="d-flex pt-4" style={{ height: '20px' }}>
```
Comment (satyamyadav) [inline core/components/atoms/collapsible/__stories__/index.story.tsx]: @AlithAnar Please avoid using style attribute, we follow BEM based css classnames and avoid inline styling. The issue seems to be due to line height mismatch for text and icon, you can add a helper class or text related class to provide the same line height as of tex to the icon (example `Text--regular`).

File: core/components/atoms/collapsible/__stories__/index.story.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ const customCode = `() => {
         height="100vh"
         onToggle={setExpanded}
       >
-        <div className="d-flex pt-4">
+        <div className="d-flex pt-4" style={{ height: "20px"}}>
```
Comment (satyamyadav) [inline core/components/atoms/collapsible/__stories__/index.story.tsx]: @AlithAnar Please avoid inline styling.

**Review by satyamyadav — CHANGES_REQUESTED** [review]

File: core/components/atoms/collapsible/__stories__/index.story.tsx (hunk 1 above)
Comment (AlithAnar) [reply core/components/atoms/collapsible/__stories__/index.story.tsx]: fixed in https://github.com/innovaccer/design-system/pull/919/commits/d23dcd332bed16a10c85c565d2b697962003b9b7

File: core/components/atoms/collapsible/__stories__/index.story.tsx (hunk 2 above)
Comment (AlithAnar) [reply core/components/atoms/collapsible/__stories__/index.story.tsx]: fixed in https://github.com/innovaccer/design-system/pull/919/commits/d23dcd332bed16a10c85c565d2b697962003b9b7

**Review by satyamyadav — APPROVED** [review]

**Review by satyamyadav — CHANGES_REQUESTED** [review]
@AlithAnar  Can you please squash the commits into one commit.

**Comment by AlithAnar** [comment]
> @AlithAnar Can you please squash the commits into one commit.

done

**Review by satyamyadav — APPROVED** [review]

**Pull request merged**